### PR TITLE
Mascara upkeep

### PR DESCRIPTION
**Pull request by frankiebee**
- [X] reload pages when switching networks
- [X] reload page if ui is not present
- [X] update cswready event

### CODE REVIEW

File: mascara/src/ui.js [hunk 1]
```diff
@@ -40,13 +40,17 @@ const connectApp = function (readSw) {
...
 background.on('ready', (sw) => {
   background.removeListener('updatefound', connectApp)
   connectApp(sw)
 })
 background.on('updatefound', () => window.location.reload())
 
 background.startWorker()
-// background.startWorker()
+.then(() => {
+  let fourceReload = setTimeout(() => {
```
Comment (kumavis) [inline mascara/src/ui.js]: unused var `fourceReload`, please remove

File: app/scripts/background.js [hunk 2]
```diff
@@ -69,6 +69,8 @@ function setupController (initState) {
     // platform specific api
     platform,
   })
+
+  controller.on('locked', platform.reload.bind(platform))
```
Comment (kumavis) [inline app/scripts/background.js]: im spinning in my grave

**Review by kumavis — CHANGES_REQUESTED** [review]

**Comment by kumavis** [comment]
generally looks good

File: app/scripts/background.js (hunk 2 above)
Comment (flyswatter) [inline app/scripts/background.js]: Do we really think this is needed? It's kinda cool that we never crash the app on purpose anymore.

Comment (frankiebee) [inline app/scripts/background.js]: i think i screwed up my git

Comment (frankiebee) [inline app/scripts/background.js]: this shouldnt be in here

**Review by kumavis — APPROVED** [review]

**Pull request merged**